### PR TITLE
Add padding to accordion content

### PR DESCRIPTION
**Pull request by MewenLeHo**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Description

Add a little bit of padding to accordion content.

### Motivation & Context

2 goals: compliance with design guidelines and prevent incoming focus of accordion title to overlap accordion content.

### Types of change

- Bug fix (non-breaking which fixes an issues)

### Live previews

* https://deploy-preview-1727--boosted.netlify.app/docs/5.2/components/accordion/

#### Contribution

- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

#### Accessibility

- [X] My change follows accessibility good practices; I have at least run axe

#### Design

- [X] My change respects the design guidelines defined in [Orange Design System](https://system.design.orange.com/0c1af118d/p/8118d1-web)
- [X] My change is compatible with responsive display

#### Development

- [X] My change follows the [developer guide](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/wiki/Developer-guide)
- (NA) I have added JavaScript unit tests to cover my changes

#### Documentation

- (NA) My change introduces changes to the documentation and/or I have updated the documentation accordingly

### Checklist (for Core Team only)

- (NA) My change introduces changes to the migration guide
- (NA) My new component is added in Storybook
- (NA) My new component is compatible with RTL
- [X] Manually run [BrowserStack tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/browserstack.yml)
- [X] Manually test browser compatibility with BrowserStack (Chrome >= 60, Firefox >= 60 (+ ESR), Edge, Safari >= 12, iOS Safari, Chrome & Firefox on Android)
- [ ] Code review
- [ ] Design review
- (NA) A11y review

#### After the merge

- [X] Manually launch [Percy tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/percy.yml)

### CODE REVIEW

File: scss/_accordion.scss [hunk 1]
```diff
@@ -85,6 +85,10 @@
   border-width: var(--#{$prefix}accordion-border-width) 0 0; // Boosted mod
 }
 
+.accordion-collapse {
+  padding: 5px 0;
```
Comment (louismaximepiton) [inline scss/_accordion.scss:89]: I think we should have `padding-top` and `padding-bottom` instead, if the user need some padding on x. Furthermore, I think that we should use a CSS variable. 
Have you tried to make the changes on `.accordion-body` (I forgot we had some already existing CSS variables)? 
Need design review?
Do we need padding on y and not only on top ?

File: scss/_accordion.scss [hunk 2]
```diff
@@ -85,6 +85,10 @@
   border-width: var(--#{$prefix}accordion-border-width) 0 0; // Boosted mod
 }
 
+.accordion-collapse {
```
Comment (louismaximepiton) [inline scss/_accordion.scss:88]: If we keep it like that, missing Boosted mod

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Few changes before approving !

File: scss/_variables.scss [hunk 3]
```diff
@@ -1545,6 +1545,7 @@ $accordion-border-color:                  $gray-500 !default; // Boosted mod
 $accordion-border-radius:                 $border-radius !default;
 $accordion-inner-border-radius:           subtract($accordion-border-radius, #{$accordion-border-width}) !default;
 
+$accordion-body-padding-top:              5px !default; // Boosted mod
```
Comment (julien-deramond) [inline scss/_variables.scss:1548]: This is not a request for a modification but rather a question on how we tackle it more globally based on Bootstrap code. Having `$accordion-body-padding-y` and `$accordion-body-padding-top` seems not very logical/practical for end-users (see https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1449/files#r970460584)

**Comment by MewenLeHo** [comment]
It seems more logical this way.
We may go further and create a new variable `$accordion-body-padding-end` even if wet set it to 0?

**Comment by MewenLeHo** [comment]
Personal reminder to update migration guide for breaking changes + new variables

File: scss/_variables.scss (hunk 3 above)
Comment (louismaximepiton) [reply scss/_variables.scss:1548]: From what was said, we should split those variables if we add a new variable: `x-padding-y` -> `x-padding-top` + `x-padding-bottom`

**Review by louismaximepiton — CHANGES_REQUESTED** [review]

**Review by louismaximepiton — APPROVED** [review]
LGTM!

**Review by julien-deramond — APPROVED** [review]
LGTM!

**Pull request merged**